client-spark/src/main/java/io/whitefox/api/client/ApiUtils.java

```diff
@@ -0,0 +1,88 @@
+package io.whitefox.api.client;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.whitefox.api.utils.ApiClient;
+import io.whitefox.api.utils.ApiException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.function.Supplier;
+
+public class ApiUtils {
+  public static <T> T recoverConflictLazy(Supplier<T> f, Supplier<T> defaultValue) {
+    try {
+      return f.get();
+    } catch (ApiException e) {
+      if (e.getCode() == 409) {
+        return defaultValue.get();
```

Comment:
I think we should do an upsert operation to avoid inconsistent read instead of returning the old object. 
Or better, we can generate unique values when creating objects to make the tests independent, especially when we will need to create multiple delta or iceberg tables.

Reply:
another solution may be to delete everything at the end of each test. But first we need to implement the delete apis.